### PR TITLE
Small docs edits

### PR DESCRIPTION
**Pull request by ElleEss**
#### Description
Small docs edits
#### Tests
Some link checking.
#### Documentation
Is docs.
#### Primary reviewers
@anne-edwards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
+Our SpatialOS community shapes the functionality of the Unreal GDK, so make sure you are part of the conversation. Come and find us on [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) or talk to us on the **Feedback** section in our [forums](https://forums.improbable.io/) (use the **unreal-gdk** tag - [this link](https://forums.improbable.io/new-topic?category=Feedback&tags=unreal-gdk) takes you there and pre-fills the category and tag). You can also [sign up for SpatialOS Unreal GDK updates](link) direct to your mailbox.
 
 #### What's the Unreal GDK all about?
-The **SpatialOS Unreal GDK** is an Unreal Engine 4 (UE4) module made by Improbable. You can use the Unreal GDK to integrate persistent multiplayer worlds into your Unreal game using the SpatialOS platform. 
+The **SpatialOS Unreal GDK** is a module for Unreal Engine 4 (UE4) made by Improbable. You can use the Unreal GDK to integrate persistent multiplayer worlds into your Unreal game using the SpatialOS platform. 
+
+For more information on the Unreal GDK and its pre-alpha release, see our [blogpost](https://improbable.io/games/blog/unreal-gdk-pre-alpha) on the Improbable website.
+
+
```
Comment (danielimprobable) [inline README.md]: Nit: newline?

File: README.md [hunk 2]
```diff
@@ -2,12 +2,15 @@
...
 
-You can [sign up for SpatialOS Unreal GDK updates](link) direct to your mailbox.
```
Comment (danielimprobable) [inline README.md]: I just got the link for this are we not having it anymore?

Comment (anne-edwards) [reply README.md]: @danielimprobable I think your changes supersede @ElleEss's changes

File: README.md (hunk 1 above)
Comment (ElleEss) [reply README.md]: Yes - to break up the text.

File: README.md (hunk 2 above)
Comment (ElleEss) [reply README.md]: Updated

**Pull request merged**